src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetEventTriggerOperation.java

```diff
@@ -91,11 +87,11 @@ public CaseEventTrigger executeForCase(String uid,
         verifyMandatoryAccessForCase(eventTriggerId, caseDetails, caseType, userRoles);
 
         return filterUpsertAccessForCase(caseType, userRoles, getEventTriggerOperation.executeForCase(uid,
-                                                       jurisdictionId,
-                                                       caseTypeId,
-                                                       caseReference,
-                                                       eventTriggerId,
-                                                       ignoreWarning));
+                                                                                                      jurisdictionId,
```

Comment:
Do we still need jurisdictionId here? I mean could we use this opportunity to clear our internal interfaces or should that be another issue?

Reply:
Yes as this one is for case not case type.

Reply:
It will likely be changed when the story to get event for case is brought up.